setup.py

```diff
@@ -0,0 +1,33 @@
+from setuptools import find_packages
+from setuptools import setup
+
+package_name = 'rpyutils'
+
+setup(
+    name=package_name,
+    version='0.1.0',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/' + package_name, ['package.xml']),
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+    ],
+    zip_safe=False,
```

Comment:
Why is it not?

Reply:
It was copied from another setup.py, removed in e5ea4f1